docs/modelview/src/tableview/tablemodel.cpp

```diff
@@ -0,0 +1,27 @@
+#include "tablemodel.h"
+
+TableModel::TableModel()
+    : QAbstractTableModel()
+{
+
+}
+
+int TableModel::rowCount(const QModelIndex &) const
+{
+    return 10;
+}
+
+int TableModel::columnCount(const QModelIndex &) const
+{
+    return 10;
+}
+
+QVariant TableModel::data(const QModelIndex &index, int role) const
+{
+    if (role == Qt::DisplayRole)
+    {
```

Comment:
no need for braces (but if you like braces that much, you should put them around the else block too, to be consistent)

Reply:
Let's drop the braces :-)